Dockerfile

```
@@ -1,55 +1,56 @@
-# Build
+################################
+# Base
+################################
 FROM node:12 AS build-deps
 LABEL maintainer="datapunt@amsterdam.nl"
 
 WORKDIR /app
 
-COPY package.json package-lock.json .env.* /app/
-COPY sitemap-generator /app/sitemap-generator
-COPY public /app/public
+COPY sitemap-generator ./sitemap-generator
 
+COPY public ./public
+
+COPY package.json package-lock.json /app/
```

Since the workdir is already the `/app` directory can we just copy these directly?

```suggestion
COPY package.json package-lock.json .
```